loadtest/store/memstore/store.go

```diff
@@ -196,18 +196,12 @@ func (s *MemStore) SetEmojis(emoji []*model.Emoji) error {
 }
 
 func (s *MemStore) SetReaction(reaction *model.Reaction) error {
-	if s.reactions[reaction.PostId] == nil {
-		s.reactions[reaction.PostId] = map[string]map[string]*model.Reaction{}
-		s.reactions[reaction.PostId][reaction.EmojiName] = map[string]*model.Reaction{}
-	} else if s.reactions[reaction.PostId][reaction.EmojiName] == nil {
-		s.reactions[reaction.PostId][reaction.EmojiName] = map[string]*model.Reaction{}
-	}
-	s.reactions[reaction.PostId][reaction.EmojiName][reaction.UserId] = reaction
+	s.reactions[reaction.PostId] = reaction
 	return nil
 }
 
 func (s *MemStore) Reaction(postId, emojiName, userId string) (*model.Reaction, error) {
-	return s.reactions[postId][emojiName][userId], nil
+	return s.reactions[postId], nil
 }
```

Comment:
This function should probably be named `Reactions` and return the slice we are storing for that `postId` which would be the only parameter.